pkg/cli/clients/authorizer_client.go

```diff
@@ -15,10 +15,11 @@ import (
 )
 
 type AuthorizerConfig struct {
-	Host     string
-	APIKey   string
-	Insecure bool
-	TenantID string
+	Host     string `flag:"host" short:"H" help:"authorizer service address" env:"TOPAZ_AUTHORIZER_SVC" default:"localhost:8282"`
+	APIKey   string `flag:"api-key" short:"k" help:"authorizer API key" env:"TOPAZ_AUTHORIZER_KEY"`
+	Token    string `flag:"token" short:"t" help:"token used for connection" env:"TOPAZ_AUTHORIZER_TOKEN"`
+	Insecure bool   `flag:"insecure" short:"i" help:"skip TLS verification"`
+	TenantID string `flag:"tenant-id" help:""`
```

Comment:
Suggest adding support for setting tenant ID using an env var, does it make sense to consistently use TOPAZ_TENANT_ID, or do we expect cross tenant operations, which I find hard to imaging the use-case for from an authorizer POV. In a directory it would be in a copy like scenario which we do not support in this CLI I believe.